frontend/src/shared/styles/workflow-node-theme.module.css

```
@@ -0,0 +1,184 @@
+  height: 80px;
+  min-width: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: var(--text-primary);
+  color: var(--bg-color);
+}
+
+.statusIdle {
+}
```

Stylelint (17.11.0)

[error] Empty block (block-no-empty)

(block-no-empty)